alpine/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM puppet/puppet-agent-alpine
+
+RUN apk add --update git \
+    && git clone https://github.com/jpogran/puppet-vscode.git
+
+ENTRYPOINT ["/usr/bin/ruby"]
+CMD ["/puppet-vscode/server/puppet-languageserver", "--no-stop", "--timeout=0"]
```

Comment:
Might be best to change the IP to any interface `--ip=0.0.0.0`  otherwise it will bind to localhost. Not sure what that is inside a container.

Reply:
Good point, will do.